app/src/ui/preferences/preferences.tsx

```diff
@@ -153,6 +161,31 @@ export class Preferences extends React.Component<
     this.props.dispatcher.removeAccount(account)
   }
 
+  private disallowedCharacterErrorMessage(name: string, email: string) {
+    const disallowedNameCharacters = disallowedCharacters(name)
+    const disallowedEmailCharacters = disallowedCharacters(email)
+
+    if (disallowedNameCharacters) {
+      return `Git name field cannot be a disallowed character "${
+        disallowedNameCharacters
+      }"`
+    } else if (disallowedEmailCharacters) {
+      return `Git email field cannot be a disallowed character "${
+        disallowedEmailCharacters
+      }"`
+    } else {
```

Comment:
We do not need the else if/if since we are returning from each branch.

Reply:
For documentation: we discussed in person and we are on board with keeping it as is since the original code shows the three returns explicitly in one chunk.